test-cases/longevity/longevity-2TB-48h-authorization-and-tls-ssl-1dis-2nondis-nemesis.yaml

```diff
@@ -26,6 +26,7 @@ n_monitor_nodes: 1
 
 instance_type_db: 'i3en.3xlarge'
 instance_type_loader: 'c5.2xlarge'
+instance_type_runner: 'r5.2xlarge'
```

Comment:
I don't mind doubling it to 4xlarge.

Reply:
I'm doing a test run with it, and we'll see